app/views/notes/_title_suggestion.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="alert alert-warning">
-	<a href="/profile/<%= user %>"><%= user %></a> is suggesting an alternative title, "<%= title %>".
-	<% if is_creator %> 
-	To accept this as a new title to your post, <a href="/node/update/title?id=<%= nid %>&title=<%= title %>" data-method='post'>click here</a>
-	<% end %> 
-</div>
\ No newline at end of file
+       <a href="/profile/<%= user %>"><%= user %></a> <%= t('notes._title_suggestion.suggesting_title') %> "<%= title %>".
+       <% if is_creator %> 
+       To accept this as a new title to your post, <a href="/node/update/title?id=<%= nid %>&title=<%= title %>" data-method='post'>click here</a>
+       <% end %> 
+</div>
```

Comment:
@Angelvicks you need to remove this file from this PR. All you have to do is undo the changes you made to this file and that should remove it. Thanks!

Reply:
@Angelvicks you were able to undo the changes but the file is still here. Do you have a formatter/linter installed? If it is disable it and then undo all of the changes again.